### PR TITLE
feat: migrate wallet creation to dashboard

### PR DESCRIPTION
**Pull request by UncleBill**

### CODE REVIEW

**Comment by guanbinrui** [comment]
@UncleBill Please fix CI errors.

File: packages/dashboard/src/pages/Wallets/CreateWallet.tsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import { useNavigate } from 'react-router'
+import { experimentalStyled as styled } from '@material-ui/core/styles'
+import { CreateWalletUI } from '../../../../maskbook/src/plugins/Wallet/UI/CreateWalletDialog'
```
Comment (septs) [inline packages/dashboard/src/pages/Wallets/CreateWallet.tsx:3]: this import path?????

Comment (Jack-Works) [reply packages/dashboard/src/pages/Wallets/CreateWallet.tsx:3]: Maybe bug of vscode

Comment (Jack-Works) [reply packages/dashboard/src/pages/Wallets/CreateWallet.tsx:3]: Please do not import `maskbook` package in the dashboard package.

Comment (UncleBill) [reply packages/dashboard/src/pages/Wallets/CreateWallet.tsx:3]: > Please do not import maskbook package in the dashboard package.

Then we might need to find a way to organize sharable UI components from plugin.

Comment (UncleBill) [reply packages/dashboard/src/pages/Wallets/CreateWallet.tsx:3]: I know we have a plan to move plugins in an individual package, maybe we can start it sooner.

File: packages/dashboard/src/pages/Wallets/CreateWallet.tsx [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import { useNavigate } from 'react-router'
+import { experimentalStyled as styled } from '@material-ui/core/styles'
+import { CreateWalletUI } from '../../../../maskbook/src/plugins/Wallet/SNSAdaptor/CreateWalletDialog'
```
Comment (Jack-Works) [inline packages/dashboard/src/pages/Wallets/CreateWallet.tsx:3]: Please don't import any value from the maskbook package, thanks.

File: packages/dashboard/src/pages/Wallets/ImportWallet.tsx [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+import { useSnackbarCallback } from '@masknet/shared'
+import { experimentalStyled as styled } from '@material-ui/core/styles'
+import {
+    ImportWalletUI,
+    RecoverResult,
+    importWallet,
+    TabIndexMap,
+    TabIndex,
+    BLANK_WORDS,
+} from '../../../../maskbook/src/plugins/Wallet/SNSAdaptor/ImportWalletDialog'
```
Comment (Jack-Works) [inline packages/dashboard/src/pages/Wallets/ImportWallet.tsx:14]: Please don't import any value from the maskbook package, thanks.

File: packages/dashboard/src/pages/Wallets/StartUp.tsx [hunk 4]
```diff
@@ -1,43 +1,52 @@
 import { experimentalStyled as styled } from '@material-ui/core/styles'
-import { StartupActionList, StartupActionListItem } from '../../components/StartupActionList'
-import { MaskWalletIcon, ImportWalletIcon, CloudLinkIcon } from '@masknet/icons'
-import { useDashboardI18N } from '../../locales'
+import { Route, Routes, useNavigate } from 'react-router'
+import { WalletCreationChooseUI } from '../../../../maskbook/src/plugins/Wallet/SNSAdaptor/CreateImportChooseDialog'
```
Comment (Jack-Works) [inline packages/dashboard/src/pages/Wallets/StartUp.tsx:3]: Please don't import any value from the maskbook package, thanks.

**Review by Jack-Works — CHANGES_REQUESTED** [review]

File: packages/dashboard/src/pages/Wallets/CreateWallet.tsx (hunk 1 above)
Comment (Jack-Works) [reply packages/dashboard/src/pages/Wallets/CreateWallet.tsx]: I'm working on this

**Review by Jack-Works — DISMISSED** [review]

**Review by Jack-Works — DISMISSED** [review]

File: packages/dashboard/src/API.tsx [hunk 5]
```diff
@@ -12,7 +12,7 @@ export interface PluginServices {
     Wallet: typeof import('../../maskbook/dist/plugins/Wallet/messages').WalletRPC
```
Comment (septs) [inline packages/dashboard/src/API.tsx:12]:
```suggestion
    Wallet: typeof import('@masknet/plugin-wallet').WalletRPC
```

Comment (septs) [reply packages/dashboard/src/API.tsx:12]: why not 
```ts
import type { WalletRPC, WalletMessages } from '@masknet/plugin-wallet'
```

Comment (Jack-Works) [reply packages/dashboard/src/API.tsx:12]: No, RPC didn't move to the `@masknet/plugin-wallet` package.

**Review by Jack-Works — APPROVED** [review]

**Comment by UncleBill** [comment]
Don't need to add wallet creation in dashboard, because this functionality will be move to popups.

Thanks @Jack-Works 

cc @nuanyang233

**Pull request closed without merging**